lib/fastlane/plugin/logme/actions/logme.rb

```diff
@@ -4,9 +4,20 @@ class LogmeAction < Action
 
       def self.run(options)
         regexp_filter = Regexp.new(options[:message_regexp_filters])
-        Actions.sh("git log --pretty=format:\"%s\" --no-merges --ancestry-path #{options[:from_revision]}...#{options[:to_revision]}")
-            .split(/\n/)
-            .select{|message| message[regexp_filter] != nil}
+        changes = Actions.sh("git log --pretty=\"%s\" --no-merges #{options[:from_revision]}...#{options[:to_revision]}")
+                      .split(/\n/)
+
+        if options[:remove_older_merge_commits]
+          merges = Actions.sh("git log --pretty=\"%b\" --merges #{options[:from_revision]}...#{options[:to_revision]}")
+                       .split("* commit")
+
+          merges.delete_at(0) # empty value
+          merges.delete_at(0) # last merge
```

Comment:
It seams, but isn't! The "* commit" is the first text that appear in the log. The split assumes that we have something in both sides. For example:
`"* commit hello".split("* commit")
 => ["", " hello"]`